client/rest.go

```diff
@@ -193,6 +193,11 @@ func (this *RestClient) sendRawTransaction(qid string, tx *types.Transaction, is
 	return this.sendRestPostRequest(common.SerializeToBytes(tx), reqPath, reqValues)
 }
 
+func (this RestClient) getStoreProof(qid string, key []byte) ([]byte, error) {
```

Comment:
`*RestClient` and  `getLayer2StoreProof`

Reply:
done